README.md

```diff
@@ -35,32 +35,108 @@ For convenience a shell script has been provided to download and extract the GOV
 ./build.sh
 ```
 
-### Configuration and Environment Variables
+### CSS / SCSS
+
+SASS is being used to build the local CSS files. To build the css files you can use npm to build by first installing the npm packages and then using:
+
+```shell
+npm run build
+```
+
+or if you'd like to watch for changes use:
+
+```shell
+npm run dev
+```
+
+To lint all CSS use:
+
+```shell
+npm run lint
+```
+
+### Set up SSL Certificate
+
+For local development we have decided to require an SSL certificate so that we run our development server with SSL so we are closer to a production system where we intend to use SSL also. We specify the flask cli flags `FLASK_RUN_CERT=cert.pem` and `FLASK_RUN_KEY=key.pem` in the `.flaskenv`, which expect a `cert.pem` and corresponding `key.pem` file in the root of the repo.
+
+You will need to create the cert-key pair with:
+
+  ```shell
+  poetry run openssl req -x509 -newkey rsa:4096 -nodes -out cert.pem -keyout key.pem -days 365
+  ```
+
+  and fill out the its prompts with information that you want (it does not matter as it is only being used for a development server).
+
+  **Note:** this command creates a cert-key pair valid for 365 days, but you can amend this as you wish.
+
+When you access the application in a new browser for the first time with one of these keys you will have to tell it you trust the certificate, but then you should not be asked again.
+
+**Note:** [flask-talisman](###HTTP-security-headers) should redirect http requests to https but whenever running the flask development server with a certificate, this doesn't seem to do the redirection. We plan to investigate but for now we will have to deal without this redirection in local dev.
+
+### Set Flask Configuration Variables
+
+Set the Flask Configuration Variables either with either:
+
+- Environment Variables:
+  1. Set all desired environment variables for all of the variables specified in `.env.env_var.template`. For convenience you can do this by running the following in the root of the repo:
+
+    ```shell
+    cp .env.env_var.template .env
```

Comment:
We're no longer going with opensearch so we could remove the empty environment variables from the env var template